### PR TITLE
Release/6.9.0

### PR DESCRIPTION
**Pull request by yardexx**
# freeRASP 6.9.0

## Pre-release checklist
- [x] 🚀 Dev/Release modes works
- [x] 🔄 Callbacks works
- [x] 📝 Logging works
  - [x] sdkVersion is correct
  - [x] sdkPlatform is correct
- [x] ✅ Tests are passing
- [x] 📋 Changelog updated
- [x] ➡️ `pubspec.yaml` version updated

## Post-release checklist
- [ ] 🐦 Release to pub.dev
- [ ] 📝 GitHub release

## Type of Changes

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Description
- New feature: ADB detection

### CODE REVIEW

File: lib/src/enums/threat.dart [hunk 1]
```diff
@@ -54,6 +54,12 @@ enum Threat {
   ///
   /// Android only
   devMode,
+
+  /// The application is running on a device that has active ADB
+  /// (Android Debug Bridge).
+  ///
+  /// Android only
+  adb,
```
Comment (tompsota) [inline lib/src/enums/threat.dart:62]: prosím pomenuj to `adbEnabled`

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.9.0] - 2024-11-19
+- Android SDK version: 13.0.0
+- iOS SDK version: 6.6.3
+
+### Flutter
+
+#### Added
+- New feature: ADB detection as a new callback for enhanced app security
```
Comment (msikyna) [inline CHANGELOG.md:15]: Added `adbEnabled` callback, which allows you to detect USB debugging option enabled in the developer settings on the device

File: example/lib/threat_notifier.dart [hunk 3]
```diff
@@ -27,6 +27,7 @@ class ThreatNotifier extends AutoDisposeNotifier<ThreatState> {
           _updateThreat(Threat.secureHardwareNotAvailable),
       onSystemVPN: () => _updateThreat(Threat.systemVPN),
       onDevMode: () => _updateThreat(Threat.devMode),
+      onADB: () => _updateThreat(Threat.adb),
```
Comment (msikyna) [inline example/lib/threat_notifier.dart:30]: onADBEnabled

File: lib/src/enums/threat.dart (hunk 1 above)
Comment (yardexx) [reply lib/src/enums/threat.dart]: Fixed.

**Review by msikyna — APPROVED** [review]

**Pull request merged**